pom.xml

```diff
@@ -74,7 +74,7 @@
     <fabric8.version>2.2.157</fabric8.version>
     <fabric8.console.version>2.2.199</fabric8.console.version>
     <fabric8.devops.version>2.2.331</fabric8.devops.version>
-    <fabric8-team-components.version>3.0.25</fabric8-team-components.version>
+    <fabric8-team-components.version>3.0.27</fabric8-team-components.version>
```

Comment:
FWIW this has already been done by the team pipeline, there's no conflict as they match but just wanted to let you know @aslakknutsen